src/App.tsx

```diff
@@ -15,48 +16,109 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  Default = 0,
+  Alphabetically = 1,
+  Length,
```

Comment:
0 is a default value for first element of enum, 1 is a second 
so 0 and 1 is redundant here 
```suggestion
  Default,
  Alphabetically,
  Length,
```